Dockerfile

```diff
@@ -1,5 +1,19 @@
 # syntax=docker/dockerfile:1
 
+# Copyright 2022 buildx-packaging
```

Comment:
Perhaps `Copyright 2022 The buildx-packaging authors` (or without year: `Copyright The buildx-packaging authors`)

Reply:
Can't remove year with https://github.com/google/addlicense. But I can change the copyright holder with `The buildx-packaging authors` if you want. Was looking at what we have in https://github.com/docker/cli-docs-tool and looks like authors is added without specifying it in copyright holder: https://github.com/docker/cli-docs-tool/blob/51b3714e65bada1083afae58281931e2610928ac/hack/license.Dockerfile#L17 :thinking:

Reply:
> looks like authors is added without specifying it in copyright holder

Nevermind it's license headers in cli-docs-tool were added manually with _authors_ string in it. Updated this PR to include _authors_.